src/constants/toast.ts

```diff
@@ -0,0 +1,23 @@
+import { Variant } from "@leafygreen-ui/toast";
```

Comment:
I wonder if it makes sense to put all of the toast code in the toasts folder. It seems pretty fragmented being in 3 locations especially if only toast is going to consume it.